powertools-validation/src/main/java/software/amazon/lambda/powertools/validation/ValidationUtils.java

```diff
@@ -270,13 +275,13 @@ private static JsonSchema createJsonSchema(String schema) {
     }
 
     private static void validateSchema(String schema, JsonSchema jsonSchema) {
-        String version = ValidationConfig.get().getSchemaVersion().toString();
+        String schemaId = ValidationConfig.get().getSchemaVersion().getId().replace("https://json-schema.org", "");
```

Comment:
What about schemas that aren't on json-schema.org?

Reply:
there are different level of schemas: 
- the schema that the user defines (to check the api gw input for example)
- the schema of the schema itself (meta schema)

We support the meta schemas 4, 6, 7, 2019, 2020, which are all on json-schema.org. And we support user schemas which are based on these meta-schemas.

there are 2 kind of validations here: 
- validating the input (using the user-defined schema)
- validating the schema itself (using the meta schema available locally in the json-schema-validator.jar or remotely on https://json-schema.org)

Reply:
Sounds good. This is the level of comprehension I didn't have time to achieve :)